test/functional/p2p_permissions.py

```diff
@@ -30,7 +30,7 @@ def run_test(self):
         self.check_tx_relay()
 
         self.checkpermission(
-            # default permissions (no specific permissions)
```

Comment:
you could do something like this instead, since this test doesn't *really* need to test rbf signaling: https://github.com/instagibbs/bitcoin/commit/a9ebb28d49769ca0f8696d32364ef1c218dc1c8c

Reply:
Agree. That patch would need some comment cleanup as well^